docs/user-guide/101-reporting.md

```diff
@@ -177,12 +177,11 @@ clutter the output with all Pod connections. This filtering can be adjusted via
 | `"{veth*,vnet*}"` | Omit all interfaces starting with either `veth` or `vnet`          |
 
 
-To override the default configuration, use `patch` command. Additionally, in
-order for the config to take effect, all handlers need to be restarted:
+To override the default configuration, use `set env` command. Additionally, in
```

Comment:
not sure I understand this: to override, don't we need to change the nmstate-handler daemonset's variable? how does `set env` do this?

Reply:
ah.. you mentioned it 2 lines below.. sorry :)